sklearn/tests/test_multioutput.py

```diff
@@ -176,6 +177,33 @@ def test_multi_output_classification_partial_fit_parallelism():
         assert est1 is not est2
 
 
+# check predict_proba passes
+def test_multi_output_predict_proba():
+    sgd_linear_clf = SGDClassifier(random_state=1, max_iter=5, tol=1e-3)
+    param = {'loss': ('hinge', 'log', 'modified_huber')}
+
+    # inner function for custom scoring
+    def custom_scorer(estimator, X, y):
+        if hasattr(estimator, "predict_proba"):
+            return 1.0
+        else:
+            return 0.0
+    grid_clf = GridSearchCV(sgd_linear_clf, param_grid=param,
+                            scoring=custom_scorer, cv=3, error_score=np.nan)
+    multi_target_linear = MultiOutputClassifier(grid_clf)
```

Comment:
I would like to `assert not hasattr(..., 'predict_proba')` before doing this fit, so that the intention of the test is a bit clearer.

Reply:
You mean for `multi_target_linear.estimator`, right? Technically, the `estimator` still wouldn't have `predict_proba` after fit because the underlying estimator (SGDClassifier with default loss='hinge') doesn't have `predict_proba`. But all estimators in `estimators_` here would (after fit, of course).

If you mean for the `multi_target_linear` itself, it _would_ have `predict_proba` before and after fit; it just won't be valid (raises ValueError)

Reply:
@jnothman thoughts?